pkg/driverbuilder/builder/oracle.go

```diff
@@ -0,0 +1,107 @@
+package builder
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+
+	"github.com/falcosecurity/driverkit/pkg/kernelrelease"
+)
+
+//go:embed templates/oracle.sh
+var oracleTemplate string
+
+// TargetTypeoracle identifies the oracle target ("ol" is the ID from /etc/os-release that Oracle uses)
+const TargetTypeoracle Type = "ol"
+
+func init() {
+	BuilderByTarget[TargetTypeoracle] = &oracle{}
+}
+
+// oracle is a driverkit target.
+type oracle struct {
+}
+
+type oracleTemplateData struct {
+	commonTemplateData
+	KernelDownloadURL string
+}
+
+func (c *oracle) Name() string {
+	return TargetTypeoracle.String()
+}
+
+func (c *oracle) TemplateScript() string {
+	return oracleTemplate
+}
+
+func (c *oracle) URLs(_ Config, kr kernelrelease.KernelRelease) ([]string, error) {
+
+	// oracle FullExtraversion looks like "-2047.510.5.5.el7uek.x86_64"
+	// need to get the "el7uek" out of the middle
+	splitVersion := strings.Split(kr.FullExtraversion, ".")
+	oracleVersion := splitVersion[len(splitVersion)-2] // [ "-2047", "510", "5", "5", "el7uek","x86_64" ] want -2
+
+	// trim off the "el" and "uek" from oracleVersion
+	version := strings.Trim(strings.Trim(oracleVersion, "el"), "uek")
+
+	// sometimes Oracle 8 does "8_x" for version, only want the "8"
+	if strings.Contains(version, "_") {
+		version = strings.Split(version, "_")[0]
+	}
+
+	// list of possible UEK versions, which are used in the URL - ex: "UEKR3"
+	// may need to evolve over time if Oracle adds more
+	ueks := []string{"R3", "R4", "R5", "R6"}
+
+	// template the kernel info into all possible URL strings
+	urls := []string{
```

Comment:
Can we add oracle9 support? 
https://github.com/falcosecurity/kernel-crawler/blob/main/kernel_crawler/oracle.py#L32

(and also `ueks R7`!)

Reply:
🤦 hahaha I totally just forgot about it. 2 sec

Reply:
Added!